tests/robustness/validate/patch_history.go

```diff
@@ -84,17 +110,25 @@ func matchWatchEvent(request *model.TxnRequest, watchEvents map[model.Event]traf
 	return nil
 }
 
-func hasNonUniqueWriteOperation(request *model.TxnRequest) bool {
-	for _, etcdOp := range request.OperationsOnSuccess {
+func canBeDiscarded(request *model.TxnRequest) bool {
+	return operationsCanBeDiscarded(request.OperationsOnSuccess) && operationsCanBeDiscarded(request.OperationsOnFailure)
```

Comment:
Is the motivation that porcupine.Operation(s) that contains `Delete and Range Type OnFailure operations` should have been discarded but not causing the robustness test timeout?

Reply:
Thanks for clarifying.